analyses/cms-open-data-ttbar/analysis.py

```diff
@@ -79,7 +79,7 @@ def parse_args() -> argparse.Namespace:
         help=(
             "Number of cores to use. In case of distributed execution this is the amount of cores per node."
         ),
-        default=len(os.sched_getaffinity(0)),
+        default=1,
```

Comment:
Why this change in particular?

Reply:
the original one doesn't work on mac (this was just changed quickly, I can add something better thought over that works everywhere)

Reply:
I can change it to: 
```
import multiprocessing
multiprocessing.cpu_count()
```